src/io/flutter/module/settings/FlutterCreateAddtionalSettingsFields.java

```diff
@@ -0,0 +1,81 @@
+
+    descriptionField = new JTextField();
+    descriptionField.setToolTipText(FlutterBundle.message("flutter.module.create.settings.description.tip"));
+    descriptionField.setText(FlutterBundle.message("flutter.module.create.settings.description.default_text"));
+
+    androidLanguageRadios = new RadiosForm(FlutterBundle.message("flutter.module.create.settings.radios.android.java"),
+                                           FlutterBundle.message("flutter.module.create.settings.radios.android.kotlin"));
+    androidLanguageRadios.setToolTipText(FlutterBundle.message("flutter.module.create.settings.radios.android.tip"));
+
+    iosLanguageRadios = new RadiosForm(FlutterBundle.message("flutter.module.create.settings.radios.ios.object_c"),
+                                       FlutterBundle.message("flutter.module.create.settings.radios.ios.swift"));
+    iosLanguageRadios.setToolTipText(FlutterBundle.message("flutter.module.create.settings.radios.ios.tip"));
+
+    includeDriverTextField = new JCheckBox();
+    includeDriverTextField.setSelected(true);
+    includeDriverTextField.setToolTipText(FlutterBundle.message("flutter.module.create.settings.includedriver.tip"));
+  }
+
+  public void addSettingsFields(@NotNull SettingsStep settingsStep) {
+    settingsStep.addSettingsField("", new JLabel(" "));
+    settingsStep.addSettingsField(FlutterBundle.message("flutter.module.create.settings.radios.type.label"),
+                                  projectTypeRadios.getComponent());
+    settingsStep.addSettingsField(FlutterBundle.message("flutter.module.create.settings.radios.org.label"), orgField);
+    settingsStep.addSettingsField(FlutterBundle.message("flutter.module.create.settings.description.label"), descriptionField);
+    settingsStep.addSettingsField(FlutterBundle.message("flutter.module.create.settings.radios.android.label"),
+                                  androidLanguageRadios.getComponent());
+    settingsStep.addSettingsField(FlutterBundle.message("flutter.module.create.settings.radios.ios.label"),
+                                  iosLanguageRadios.getComponent());
+    settingsStep.addSettingsField(FlutterBundle.message("flutter.module.create.settings.includedriver.label"), includeDriverTextField);
+
+    settingsStep.addSettingsField("", new JLabel(" "));
+    settingsStep.addSettingsField(FlutterBundle.message("flutter.module.create.settings.help.label"), new JLabel(" "));
+    settingsStep.addSettingsField(FlutterBundle.message("flutter.module.create.settings.help.project_name.label"),
+                                  new JLabel(FlutterBundle.message("flutter.module.create.settings.help.project_name.description")));
+    settingsStep.addSettingsField(FlutterBundle.message("flutter.module.create.settings.help.org.label"),
+                                  new JLabel(FlutterBundle.message("flutter.module.create.settings.help.org.description")));
```

Comment:
Add help for project type, to help distinguish Application vs plugin. Something 

Project type: Select application to build a mobile application for end users.
                      Choose a plugin if you're building native device feature hooks for a developer.